src/Function/Consequences.agda

```diff
@@ -112,3 +113,44 @@ strictlyInverseʳ⇒inverseʳ : Transitive ≈₁ →
                             Inverseʳ ≈₁ ≈₂ f f⁻¹
 strictlyInverseʳ⇒inverseʳ trans cong sinv {x} y≈f⁻¹x =
   trans (cong y≈f⁻¹x) (sinv x)
+
+------------------------------------------------------------------------
+-- Theory of the section of a Surjective function
+
+module Section (≈₂ : Rel B ℓ₂) (surj :  Surjective {A = A} ≈₁ ≈₂ f) where
+
+  section : B → A
+  section = proj₁ ∘ surj
+
+  inverseˡ : Inverseˡ ≈₁ ≈₂ f section
+  inverseˡ {x = x} = proj₂ (surj x)
+
+  module _ (refl : Reflexive ≈₁) where
+
+    strictlySurjective : StrictlySurjective ≈₂ f
+    strictlySurjective = surjective⇒strictlySurjective ≈₂ refl surj
+
+    strictlyInverseˡ : StrictlyInverseˡ ≈₂ f section
+    strictlyInverseˡ _ = inverseˡ refl
+
+    strictlyInverseʳ : StrictlyInverseʳ ≈₂ section f
+    strictlyInverseʳ _ = inverseˡ refl
```

Comment:
Is this the right definition to export?
Or do we want, later, to have `strictlyInverseʳ : StrictlyInverseʳ ≈₂ f section` as a consequence of `inverseʳ : Transitive ≈₂ → Inverseʳ ≈₁ ≈₂ f section`?